src/common/envalid.ts

```diff
@@ -32,6 +32,8 @@ export const cliEnvSpec = {
   RANDOM: bool({ desc: 'Randomizes the timeouts by multiplying with a factor between 1 to 2', default: false }),
   MIN_TIMEOUT: num({ desc: 'The number of milliseconds before starting the first retry', default: 60000 }),
   FACTOR: num({ desc: 'The factor to multiply the timeout with', default: 1 }),
+  GITEA_URL: str({ default: 'gitea-http.gitea.svc.cluster.local:3000' }),
```

Comment:
GITEA_URL could be split into:
GITEA_HOST and GITEA_PORT

Reply:
Perhaps also instead of GITEA we could use GIT because the operator is git repo provider agnostic